pkgs/tools/audio/tts/default.nix

```diff
@@ -88,17 +83,23 @@ python3Packages.buildPythonApplication rec {
     tqdm
     librosa
     unidecode
+    umap-learn
     phonemizer
     tensorboardx
     fuzzywuzzy
-    tensorflow_2
     inflect
     gdown
     pysbd
+    pyworld
   ];
 
   postInstall = ''
     cp -r TTS/server/templates/ $out/${python3.sitePackages}/TTS/server
+    # cython modules are not installed for some reasons
+    (
+      cd TTS/tts/layers/glow_tts/monotonic_align
+      ${python3Packages.python.interpreter} setup.py install --prefix=$out
+    )
```

Comment:
I am not sure if the subshell could break things. Does it forward everything correctly?

Reply:
Yes: 

```
$ bash -c '(false)'; echo $?
1
```